packages/shared/create-local-vue.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import Vue from 'vue'
+import cloneDeep from 'lodash/cloneDeep'
+
+/**
+ * Used internally by vue-server-test-utils and test-utils to propagate/create vue instances.
+ * This method is wrapped by createLocalVue in test-utils to provide a different public API signature
+ * @param {Component} _Vue
+ * @param {VueConfig} config
+ * @returns {Component}
+ */
+function _createLocalVue(
+  _Vue: Component = Vue,
+  config: VueConfig = {}
+): Component {
+  const instance = _Vue.extend()
+
+  // clone global APIs
+  Object.keys(_Vue).forEach(key => {
+    if (!instance.hasOwnProperty(key)) {
+      const original = _Vue[key]
+      // cloneDeep can fail when cloning Vue instances
+      // cloneDeep checks that the instance has a Symbol
+      // which errors in Vue < 2.17 (https://github.com/vuejs/vue/pull/7878)
+      try {
+        instance[key] =
+          typeof original === 'object' ? cloneDeep(original) : original
+      } catch (e) {
+        instance[key] = original
+      }
+    }
+  })
+
+  // config is not enumerable
+  instance.config = cloneDeep(Vue.config)
+
+  // if a user defined errorHandler is defined by a localVue instance via createLocalVue, register it
+  instance.config.errorHandler = config.errorHandler || Vue.config.errorHandler
```

Comment:
This is the only line that has changed within the logic, IE adding `config.errorHandler`. If we do not feel confortable reviewing this due to scope of change, we can break out the `shared` move in a separate PR to more clearly see the change set

Reply:
shared is private anyway, this should be fine